mongodb/mongo-init.js

```diff
@@ -0,0 +1,908 @@
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000006',
+                eventCode: 'TRANSACTION_USER_CANCELED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000003-0000-0000-0000-000000000006',
+                eventCode: 'TRANSACTION_CLOSURE_ERROR_EVENT',
+                creationDate: "2025-01-13T09:18:18.000000000Z[Etc/UTC]"
+            },
+        ]
+    },
+    // 7 - TransactionExpirationQueueConsumerV2 - ACTIVATED
+    {
+        transactionId: "00000000000000000000000000000007",
+        status: 'ACTIVATED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000007',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+        ]
+    },
+    // 8 - TransactionExpirationQueueConsumerV2 - AUTHORIZATION_REQUESTED
+    {
+        transactionId: "00000000000000000000000000000008",
+        status: 'AUTHORIZATION_REQUESTED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000008',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000008',
+                eventCode: 'TRANSACTION_AUTHORIZATION_REQUESTED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            },
+        ]
+    },
+    // 9 - TransactionExpirationQueueConsumerV2 - AUTHORIZATION_COMPLETED
+    {
+        transactionId: "00000000000000000000000000000009",
+        status: 'AUTHORIZATION_COMPLETED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000009',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000009',
+                eventCode: 'TRANSACTION_AUTHORIZATION_REQUESTED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000003-0000-0000-0000-000000000009',
+                eventCode: 'TRANSACTION_AUTHORIZATION_COMPLETED_EVENT',
+                creationDate: "2025-01-13T09:18:18.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000004-0000-0000-0000-000000000009',
+                eventCode: 'TRANSACTION_REFUND_REQUESTED_EVENT',
+                creationDate: "2025-01-13T09:18:19.000000000Z[Etc/UTC]"
+            },
```

Comment:
```suggestion
            },
```

for have a transaction in CLOSURE_REQUESTED state this event have to be remove